ironfish-http-api-v2/src/main.ts

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { NestFactory } from '@nestjs/core'
+import { ExpressAdapter, NestExpressApplication } from '@nestjs/platform-express'
+import compression from 'compression'
+import express from 'express'
+import helmet from 'helmet'
+import http from 'http'
+import { AppModule } from './app.module'
+
+const PORT = process.env.PORT || 8003
```

Comment:
It would be nice if it printed the port out in the logs when it started listening, small detail but useful when there's many projects who use different ports.

Reply:
Sounds good - I added a Linear ticket for this